### PR TITLE
added some Icons

### PR DESCRIPTION
**Pull request by Dreamtrailtv**

### CODE REVIEW

File: customizations/Microsoft Office.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "color": "#D83B01",
+    "imageUrl": "https://i.imgur.com/3KCsuIH.png",
```
Comment (vascYT) [inline customizations/Microsoft Office.json:3]: The icon is too small.

**Comment by vascYT** [comment]
Please send me a reference link to the "Skyblock Dungeons Guide"

**Comment by Dreamtrailtv** [comment]
image
https://github.com/Dungeons-Guide/Skyblock-Dungeons-Guide

File: customizations/Microsoft Office.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "color": "#D83B01",
+    "imageUrl": "https://i.imgur.com/eykwrB5.png",
```
Comment (vascYT) [inline customizations/Microsoft Office.json:3]: crop of the blank space around the logo

**Review by vascYT — APPROVED** [review]

**Pull request merged**